flocker/provision/_install.py

```diff
@@ -93,8 +126,12 @@ def install_cli_commands_yum(distribution, package_source):
         base_url = urljoin(package_source.build_server, result_path)
     else:
         use_development_branch = False
+    # This references CentOS 7 but this only works because CentOS 7's
+    # repository URL is the same as Fedora 20's.
     commands = [
-        sudo(command="yum install -y " + CLUSTERHQ_REPO[distribution])
+        sudo(command="yum install -y " + get_repository_url(
+            distribution='centos-7',
```

Comment:
Why are you not using `distribution=distribution` here?  The repo URL for fedora and CentOS is different: one contains `centos` and the other contains `fedora`.